Cannot compile - Error regarding Explorer::Set_backlight

Function pico_explorer::set_backlight(100) on line 14 of main.cpp causes compilation error.

/mnt/c/Users/James/Documents/Programming/pico-dev/pico-test/main.cpp: In function 'int main()':
/mnt/c/Users/James/Documents/Programming/pico-dev/pico-test/main.cpp:14:19: error: 'class pimoroni::PicoExplorer' has no member named 'set_backlight'
   14 |     pico_explorer.set_backlight(100);
      |                   ^~~~~~~~~~~~~
make[2]: *** [CMakeFiles/simple_test.dir/build.make:63: CMakeFiles/simple_test.dir/main.cpp.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:1535: CMakeFiles/simple_test.dir/all] Error 2
make: *** [Makefile:152: all] Error 2

--- a/main.cpp
+++ b/main.cpp
@@ -8,11 +8,6 @@ PicoExplorer pico_explorer(buffer);
 int main() {
     pico_explorer.init();
 
-    // set the backlight to a value between 0 and 255
-    // the backlight is driven via PWM and is gamma corrected by our
-    // library to give a gorgeous linear brightness range.
-    pico_explorer.set_backlight(100);
-
     pico_explorer.set_pen(255, 0, 0);
 
     while(true) {

